src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/GetPileupSummaries.java

```diff
@@ -54,10 +54,10 @@
 @DocumentedFeature
 public class GetPileupSummaries extends MultiVariantWalker {
 
-    public final String MAX_SITE_AF_LONG_NAME = "maximumPopulationAlleleFrequency";
-    public final String MIN_SITE_AF_LONG_NAME = "minimumPopulationAlleleFrequency";
-    public final String MAX_SITE_AF_SHORT_NAME = "maxAF";
-    public final String MIN_SITE_AF_SHORT_NAME = "minAF";
+    public static final String MAX_SITE_AF_LONG_NAME = "maximum-population-allele-frequency";
```

Comment:
- Update program group VariantProgramGroup --> CoverageAnalysisProgramGroup.java in the import statement and for CommandLineProgramProperties.
- Update `gatk-launch --javaOptions "-Xmx4g"` --> `gatk`
- I think it's important to clarify that the full gnomAD resource (an unwieldy file) isn't what is required per se but a VCF of commonly variant germline sites containing population allele frequencies is what is required. Didn't you tell me once that this file MUST only contain one ALT allele per record? If this is true, mentioning it is vital as gnomAD itself certainly cannot be described as such. Another thing to include is that "An eight-column sites-only VCF is acceptable. See the GATK Resource Bundle for an example human file."

Reply:
done x 3